alpine/Dockerfile

```diff
@@ -33,7 +33,7 @@ ADD depends /depends
 RUN cd /depends && ./install_webp.sh && ./install_imagequant.sh && ./install_raqm.sh
 
 RUN /usr/sbin/adduser -D pillow && \
-    pip3 install virtualenv && virtualenv /vpy3 && \
+    pip3 install -I virtualenv && virtualenv /vpy3 && \
```

Comment:
```
    -I                    don't match patterns in binary files
```

What exactly is this doing?

Reply:
The longer version of the flag is '--ignore-installed' - https://stackoverflow.com/q/51913361/4093019

Reply:
Ah, thanks. The above was the help text for grep, I searched my console for `-i ` and got the wrong one!

Real one:

```
  -I, --ignore-installed      Ignore the installed packages, overwriting them. This can break your system if the existing package is
                              of a different version or was installed with a different package manager!
```